.nvmrc

```diff
@@ -0,0 +1 @@
+22
```

Comment:
I'd generate this file with `node -v > .nvmrc`

Reply:
Well, I did not use that to avoid using a specific version and use the latest node 22 version.